NuGet.config

```diff
@@ -16,7 +16,9 @@
         <add key="repositoryPath" value="..\packages" />
     </config>
   <packageSources>
+    <add key="LocallyBuiltApplicationInsightsPackages" value="..\bin\Debug\NuGet" />
```

Comment:
If that test app is in the same solution then can you do a project reference instead of referencing the dropped package?